module_utils/network_lsr/argument_validator.py

```diff
@@ -896,6 +896,14 @@ def _validate_post_fields(self, value, name, result):
         state = result.get("state")
         persistent_state = result.get("persistent_state")
 
+        # if persistent state is present state cannot be set to present or absent
+        if persistent_state == "present" and state:
```

Comment:
Please check if persistent state is present or absent. Also, this can be one if statement like:

`if persistent_state in ... and state in ...:`

Reply:
Fixed, thanks!